root/meta/genreflex/ROOT-5768/RelationalCool/scripts/coolSqlplus.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 # See https://twiki.cern.ch/twiki/bin/view/PSSGroup/RlWrap
 if [ "${SCRAM_ARCH:0:5}" = "win32" ]; then
```

Comment:
I am guessing that this needs to be checked ...

Reply:
@pcanal yes, I missed that one, pushing fix...